recipes/ggobi/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set version = '2.1.11' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-rggobi
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url: http://www.ggobi.org/downloads/ggobi-2.1.11.tar.bz2
+  sha256: 2c4ddc3ab71877ba184523e47b0637526e6f3701bd9afb6472e6dfc25646aed7
+  patches:
+    - osx_patch  # [osx]
+
+build:
+  skip: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - autoconf
+    - automake
+    - pkg-config
+    - make
+  host:
+    - gtk2
+    - libxml2
+  run:
+    - gtk2
+    - libxml2
+
+test:
+  # Functional tests require a running X server
```

Comment:
this can now be removed?

Reply:
It can, I just left it in there for people wondering why the test was wonky.